Content.Server/Cargo/CargoOrderDatabase.cs

```diff
@@ -86,18 +86,23 @@ public bool RemoveOrder(int orderNumber)
         /// <param name="order">The order to be approved.</param>
         public bool ApproveOrder(int orderNumber)
         {
-            if (CurrentOrderSize == MaxOrderSize)
-                return false;
-            if (!_orders.TryGetValue(orderNumber, out var order))
-                return false;
-            if (order.Approved)
+            if (CurrentOrderSize == MaxOrderSize ||
+                !_orders.TryGetValue(orderNumber, out var order) ||
+                order.Approved)
+            {
                 return false;
+            }
             else if (CurrentOrderSize + order.Amount > MaxOrderSize)
             {
-                AddOrder(order.Requester, Loc.GetString("{0} (Overflow)", order.Reason.Replace(" (Overflow)", "")), order.ProductId,
+                AddOrder(
+                    order.Requester,
+                    Loc.GetString("cargo-order-database-order-overflow-message", ("placeholder", order.Reason.Replace(" (Overflow)", string.Empty))),
```

Comment:
Can't `.Replace()` localised text like this. `(Overflow)` is english, and thus probably won't be present in localised versions of this feature!

Reply:
I think Replace() is for order.Reason. It takes the reason (which doesn't seem to have any tokens at all), and removes " (Overflow)" from it, then puts the reason  into {placeHolder} in the token.